persistence/src/main/java/org/mskcc/cmo/metadb/persistence/CmoRequestRepository.java

```diff
@@ -13,18 +13,18 @@
  * @author ochoaa
  */
 @Repository
-public interface CmoRequestRepository extends Neo4jRepository<CmoRequestEntity, Long> {
-    @Query("MATCH (r: cmo_metadb_request {requestId: $reqId}) RETURN r;")
-    CmoRequestEntity findByRequestId(@Param("reqId") String reqId);
+public interface CmoRequestRepository extends Neo4jRepository<MetaDbRequest, Long> {
+    @Query("MATCH (r: MetaDbRequest{requestId: $reqId}) RETURN r;")
```

Comment:
would you mind making the styling consistent with the spacing before brackets, etc.?